.gitignore

```diff
@@ -12,6 +12,7 @@
 .\#*\#
 .vs
 .clang-format
```

Comment:
instead of ignoring .clang-format, it could be useful to track this file here. Thisis due to how clang-format work. When launched without other arguments than files, clang-format search the directories up until .clang-format is found. Tracking this file here instead of in the 'script' subdir will ensure the right configuration file is used when formating files.

Reply:
I dont't know about tracking `.clang-format`, but we need to track the script directory anyway.

Reply:
Sure, the script directory must be tracked.
But, to ease IDE integration, having the file `.clang-format`at the root of the Radium dir could be helpful.
From the `clang-format --help`
```
  -style=<string>           - Coding style, currently supports:
LLVM, Google, Chromium, Mozilla, WebKit.
    Use -style=file to load style configuration from
    .clang-format file located in one of the parent
    directories of the source file (or current
    directory for stdin).
    Use -style="{key: value, ...}" to set specific
    parameters, e.g.:
        -style="{BasedOnStyle: llvm, IndentWidth: 8}"
```
Note that the `.clang-format` file is search by default if nothing is given using this option.

Reply:
well, I have a symlink `.clang-format -> scripts/clang-format`
and  afaik symlink are not a so good idea in git repo (some windows file system do not support them)
I think it's user responsability to install the script, with `scripts/install-scripts-linux.sh` for instance